LiskHQ/lisk
# Unauthorized try of change account's virgin field from 0 to 1 - Closes #1504

### What was the problem?
Whenever we try to update any account for any attribute it was including **virgin** in update SQL. We have a DB trigger that protects any change in **virgin** attribute once its set to 0. Due to that unwanted update for **virgin** attribute we were having log message from database. 

### How did I fix it?
I found the occurrence of the code where we actually convert an account to non-vrigin explicitly 

https://github.com/LiskHQ/lisk/blob/7da23971ae74a6f57ba274c215efbb6e06da5817/logic/account.js#L921-L923

And converted this call to a special method in db repo for account and removed virgin attribute from update column set. 

### How to test it?

Start the app, while applying genesis block, look for warning messages 
> WARNING:  Reverting change of virgin from 0 to 1

If you don't see this warning then the issue would be fixed 😄 

### Review checklist

* The PR solves #1504
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
